main.js

```diff
@@ -0,0 +1,43 @@
+// Task 1
+
+let user = 'John Doe';
+console.log(user);
+let student ='Masha';
+console.log(student);
+user = student;
+console.log(user); //Masha
+
+//Task 2
+let test = 1;
+test += 1; 
+test += '1';
+console.log(test); //21
+test -= 1;
+console.log(test); //20
+test = Boolean(test); //true
+console.log(test);
+
+//Task 3
+let arr = [2, 3, 5, 8];
+let newArr = 1;
```

Comment:
That doesn't arr, change the name like `result`.

Reply:
ok, boss